container_cloud_config/templates/dockersystemd.yaml

```diff
@@ -24,6 +26,9 @@
     {%- endif %}
     {% for after in after_units -%}
     After={{ after }}
+    {% if not oneshot -%}
```

Comment:
This should be wrapped around the `ExecStop=docker stop` instruction below. After and Requires should be split out into separate APIs